player/js/worker_wrapper.js

```diff
@@ -687,10 +687,14 @@ var lottie = (function () {
         });
       },
       resize: function () {
+        var devicePixelRatio = window.devicePixelRatio || 1;
         workerInstance.postMessage({
           type: 'resize',
           payload: {
             id: animationId,
+            // Till Worker thread knows nothing about container, we've to pass it here
+            width: animation.container.offsetWidth * devicePixelRatio,
```

Comment:
animation.container could be undefined.
what about changing it to:
`width: width || (animation.container ? animation.container.offsetWidth * devicePixelRatio : 0),
height: height || (animation.container ? animation.container.offsetHeight * devicePixelRatio : 0),
`  
and letting the resize method accept 2 arguments `width` and `height`?

Reply:
> animation.container could be undefined. what about changing it to: `width: width || (animation.container ? animation.container.offsetWidth * devicePixelRatio : 0), height: height || (animation.container ? animation.container.offsetHeight * devicePixelRatio : 0), ` and letting the resize method accept 2 arguments `width` and `height`?

Yeah, It should definitely accept `width` and `height` as params